src/modules/map/components/toolbox/TimeSlider.vue

```diff
@@ -152,13 +177,37 @@ onMounted(() => {
     } else {
         currentYear.value = previewYear.value
     }
+    tippyInstance = tippy(yearCursorInput.value, {
+        content: tooltipContent,
+        arrow: true,
+        hideOnClick: false,
+        placement: 'bottom',
+        trigger: 'manual',
+        theme: 'danger',
+    })
 })
 
 onUnmounted(() => {
     // TODO : when we have an 'activeTimeSlider' in store, we'll get rid of this.
     store.dispatch('clearPreviewYear', dispatcher)
+    tippyInstance.destroy()
 })
 
+function tooltipContent() {
+    return (
+        i18n.t('outside_valid_year_range') +
+        '{' +
+        ALL_YEARS[0] +
+        '-' +
+        ALL_YEARS[ALL_YEARS.length - 1] +
+        '}'
```

Comment:
Perso I would not use the `{}` simply

```javascript
return `${i18n.t('outside_valid_year_range')} ${ALL_YEARS[0]}-${ALL_YEARS[ALL_YEARS.length - 1]}`
```

The output will look nicer less coding.